applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/track_interface.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+'require view';
+'require form';
+'require tools.widgets as widgets';
+'require uci';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('keepalived', _('Track Interface'));
+
+		s = m.section(form.GridSection, 'track_interface');
+		s.anonymous = true;
+		s.addremove = true;
+		s.nodescriptions = true;
+
+		o = s.option(form.Value, 'name', _('Name'));
+		o.rmempty = false;
+		o.optional = false;
+		o.placeholder = 'name';
```

Comment:
please Remove this placeholder text it does not help

Reply:
Done